chart/bagetter/README.md

```diff
@@ -0,0 +1,36 @@
+# bagetter
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.4](https://img.shields.io/badge/AppVersion-1.0.4-informational?style=flat-square)
```

Comment:
AppVersion is here static, too.

Reply:
This file was generated by `helm-docs`
Should be re-generated by `release.yml`, too